apps/discovery_streams/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule DiscoveryStreams.Mixfile do
   def project do
     [
       app: :discovery_streams,
-      version: "2.10.0",
+      version: "2.10.1",
```

Comment:
There are definitely some cases in this app that I know of where `:discovery_streams` is hardcoded in the `Brook.Event.send` and other functions. It feels like we should at least take care of those too.